timescaledb/tutorials/nfl-fantasy-league.md

```diff
@@ -171,44 +202,147 @@ conn.close()
+SELECT t.player_id, t.gameid, 
+	time_bucket(INTERVAL '1 day', t."time") AS bucket,
+	SUM(t.dis) AS yards
+FROM tracking t 
+GROUP BY t.player_id, t.gameid, bucket;
 ```
 
-1. Number of yards run in game for passing plays, by player and game 
-1. Avg yards run for a player over a game
-1. Average and median yards run per game by type of player (not taking avg of individual)
-1. Num of snap plays by player where they were on the offense
-1. [Number of plays vs points scored](number-of-players-vs-points-scored)
+1. [Number of yards run in game for passing plays, by player and game](#number-of-yards-run-in-game-for-passing-plays-by-player-and-game)
+1. [Average yards run for a player over a game](#average-yards-run-for-a-player-over-a-game)
+1. [Average and median yards run per game by type of player (not taking avg of individual)](#average-and-median-yards-run-per-game-by-type-of-player-not-taking-avg-of-individual)
+1. [Number of snap plays by player where they were on the offense](#number-of-snap-plays-by-player-where-they-were-on-the-offense)
+1. [Number of plays vs points scored](#number-of-plays-vs-points-scored)
 1. [Average yards per game for top three players of each position](#average-yards-per-game-for-top-three-players-of-each-position)
 
+### **Number of yards run in game for passing plays, by player and game**
+
+To get this data, use the following query which pulls yard data from the continuous aggregate and then joins that on the player table to get player details. 
+
+```sql
+SELECT a.player_id, display_name, SUM(yards) AS yards, gameid 
+FROM player_yards_by_game a
+LEFT JOIN player p ON a.player_id = p.player_id 
+GROUP BY a.player_id, display_name, gameid 
+ORDER BY gameid ASC, display_name
+```
+Your data will look like this:
+
+|player_id| display_name        | yards           | gameid  |
+|-----| ------------- |:-------------:| -----:|
+|2555415| Austin Hooper     | 765.52 | 2018090600 |
+|2556445| Brian Poole    | 661.74     |   2018090600 |
+|2560854| Calvin Ridley | 822.3     |    2018090600 |
+
+This query can be the foundation of many other analysis questions. In the following queries, we will be returning to the query for further analysis. 
+
+### **Average yards run for a player over a game**
+
+This query uses one of timescale's percentile functions to find the mean yards run per game by a single player. 
+
+```sql
+WITH sum_yards AS (
+  SELECT a.player_id, display_name, SUM(yards) AS yards, gameid 
+  FROM player_yards_by_game a
+  LEFT JOIN player p ON a.player_id = p.player_id 
+  GROUP BY a.player_id, display_name, gameid 
+)
+SELECT player_id, display_name, mean(percentile_agg(yards)) as yards
+FROM sum_yards
+GROUP BY player_id, display_name
+ORDER BY yards DESC
+```
+After running this query you will notice the player_id and display_name are null for the first row. Can you guess what this row of data means? What could the average number of yards run mean for the top players listed?
```

Comment:
```suggestion
When you run this query you might notice the `player_id` and `display_name` are null for the first row. Can you guess what this row of data means? What could the average number of yards run mean for the top players listed?
```
And reflow

I'm not sure about the hypothetical questions here. Can we just declare why it's so instead?